### DISCUSSION TITLE
I have node version 14.x and Also Python 3.7.x, but the installation is not getting completed

### DISCUSSION THREAD

**Discussion by ashutoshsinghai** [strapi/strapi#7387] [Legacy Topics]
Even though I have the newer version, it asks for node 12
Installation is getting hanged
Some dependency is creating problem

Please suggest all the prerequisites for installing strapi

**Comment by derrickmehaffy** [comment]
@ashutoshsinghai I had planned on doing a PR to the `README.md` to specify node v12 instead of >= 10.

You are correct in that Strapi does not support node v14 as that version doesn't go LTS until October, and Strapi only supports LTS versions of node (v10/v12)

**Reply by ashutoshsinghai** [reply]
That's fine 
But what with these logs ?
some of them says, required node version <= 8
![image](https://user-images.githubusercontent.com/42891427/89806718-23f23380-db55-11ea-8151-c045b4c7600f.png)

**Reply by derrickmehaffy** [reply]
It would appear that is a depends of `webpack` based on digging through a fresh v3.1.3 yarn.lock file but I have no issues running it on Node v12. I would say that specific error can be safely ignored.
